<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExampleClass</class>
 <widget class="QDialog" name="ExampleClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>317</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Example</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="chk_underline_">
        <property name="text">
         <string>Underline</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chk_italic_">
        <property name="text">
         <string>Italic</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chk_bold_">
        <property name="text">
         <string>Bold</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="rbtn_black_">
        <property name="text">
         <string>Black</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbtn_red_">
        <property name="text">
         <string>Red</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbtn_blue_">
        <property name="text">
         <string>Blue</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPlainTextEdit" name="edit_text_">
       <property name="plainText">
        <string>Hello.world
It is my demo.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_ok_">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel_">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_exit_">
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="example.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>OnBtnExitClicked()</slot>
  <slot>OnBtnCancelClicked()</slot>
  <slot>OnChkUnderlineClicked(bool)</slot>
 </slots>
</ui>
